manifests/cf-rabbitmq-server-only-template.yml

```diff
@@ -46,6 +46,8 @@ properties:
     - rabbitmq_management
     - rabbitmq_mqtt
     - rabbitmq_stomp
+    policies:
+      HA: '''^(?!amq\.).*'' ''{"ha-mode": "all"}'''
```

Comment:
@michaelklishin thank you for suggestion.
I've just amended template file, could you please review.